velox/docs/functions/presto/decimal.rst

```diff
@@ -150,8 +150,28 @@ Decimal Functions
         p = min(38, p1 + s2 + max(0, s2 - s1))
         s = max(s1, s2)
 
-    Throws if y is zero or result cannot be represented using precision calculated
+    Throws if:
+    1. y is zero or result cannot be represented using precision calculated
     above.
+    2. max(s1, s2) - s1 + s2 exceeds 38.
+
+    Let’s take a closer look at how division is implemented.
```

Comment:
Would this belong to Division section above? L160?

Would you generate the docs locally to confirm they look nice?

Reply:
I verified the doc in clion:

<img width="1372" alt="Screenshot 2024-04-05 at 4 35 17 PM" src="https://github.com/facebookincubator/velox/assets/6971561/651815db-0001-4663-8483-f370fc17d057">

Reply:
@karteekmurthys Karteek, to verify the docs it is necessary to generate these locally and view them in the browser.